packages/webdriverio/src/test.ts

```diff
@@ -118,6 +71,10 @@ describe('@axe-core/webdriverio', () => {
       server.close();
     });
     describe('AxeBuilder', () => {
+      it('check to make sure that client is running devtools protocol', () => {
+        assert.isTrue(client.isDevTools);
```

Comment:
I was under the impression you were going to test both webdriver and devtools. I feel like only testing one is a mistake.